packages/modules/web-viewer-react/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext"
   },
   "include": ["./src/**/*.ts*"],
```

Comment:
can you try to exclude all test files? Vitest shouldn't need a lib output to run the tests (compared to jest). We can reduce the file size in `lib` then

Reply:
reducing file size in lib will make no difference, we dont ship the entirety of lib. see `.npmignore`.
the only benefit here would be saving a few seconds during compilation.